enterprise_access/apps/api/tests/test_tasks.py

```diff
@@ -129,12 +134,19 @@ def test_send_notification_emails_for_requests(self, mock_braze_client, mock_lms
                 'alias_name': user_email,
             },
         }
-        mock_braze_client().send_campaign_message.assert_called_with(
+        expected_course_about_page_url = (
+            'http://enterprise-learner-portal.example.com/test-org-for-learning/course/' +
+            self.license_requests[0].course_id
+        )
```

Comment:
nit: Would this make sense as an `f` string, e.g. `f'http://enterprise-learner-portal.example.com/test-org-for-learning/course/{self.license_requests[0].course_id}'`?

Reply:
I split this up because the line was too long, I think either way is fine? I can combine 2 f strings together though

Reply:
Twas a long line, which is why I split it up this way